### PR TITLE
LINK-1432 | Update signup group page

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Start using new `PUT /signup_group/<id>` endpoint to update signup group and all the signups it contains.
pdate
Move u signup page to route `/registration/<id>/signup-group/<id>/edit`

Open edit signup group page when clicking signup with `signup_group` in signup list page. If signup doesn't have `signup_group` show toast message. There will be separate PR to implement page to update a single singup

Move components and functions related to signup group update to signup-group folder

https://github.com/City-of-Helsinki/linkedevents/pull/680 should be merged before this PR to avoid conflicts with API.

## Partially closes
[LINK-1432](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1432)

[LINK-1432]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1432?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  const goBack = useGoBack<EnrolmentsLocationState>({
+    defaultReturnPath: ROUTES.REGISTRATION_ENROLMENTS.replace(
+      ':registrationId',
+      getValue(registration.id, '')
+    ),
+    state: { enrolmentId: responsiblePerson?.id },
+  });
+
+  const getSingupActionItemProps = ({
```
Comment (mikkojamG) [inline src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx:62]: Small typo here ☝️

File: src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      authenticated,
+      onClick,
+      organizationAncestors,
+      publisher,
+      t,
+      user,
+    });
+  };
+
+  const getSingupGroupActionItemProps = ({
```
Comment (mikkojamG) [inline src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx:80]: Small typo here ☝️

**Review by mikkojamG — APPROVED** [review]
Spotted couple of typos, otherwise LGTM 👍

File: src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/signupGroup/editButtonPanel/EditSignupGroupButtonPanel.tsx]: Great catch from such huge amount of lines

**Pull request merged**